assets/lib/dialog/dialog.js

```diff
@@ -104,12 +104,9 @@
 		};
 
 		this.maybeLoadAssets = async function () {
-			const isWpAdmin = !! document.querySelector( 'body.wp-admin' ),
-				isElementorEditor = !! document.querySelector( 'body.elementor-editor-active' ),
-				isFrontend = ! isWpAdmin && ! isElementorEditor,
-				hasAdminBar = !! document.querySelector( '#wpadminbar' );
+			const isFrontend = window.hasOwnProperty( 'elementorFrontend' ) && window.elementorFrontend.hasOwnProperty( 'utils' ) && window.elementorFrontend.utils.hasOwnProperty( 'assetsLoader' )
```

Comment:
I think this is the safest way to verify that the `elementorFrontend` localization object and the `assetsLoader` object are present.